src/QsCompiler/Transformations/Utils.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Quantum.QsCompiler.Transformations
+{
+    /// <summary>
+    /// Helpful utility functions.
+    /// </summary>
+    public static class Utils
```

Comment:
@bettinaheim, can you review these additions to the public API?

Reply:
Based on our discussion, I'm keeping these utilities internal and using `InternalsVisibleTo` to share them with the CompilationManager project.